cmd/broker/broker_suite_test.go

```diff
@@ -918,70 +688,36 @@ func (s *BrokerSuiteTest) AssertDisabledNetworkFilterRuntimeState(runtimeid, op
 	require.NoError(s.t, err)
 }
 
-func (s *BrokerSuiteTest) LastProvisionInput(iid string) gqlschema.ProvisionRuntimeInput {
-	// wait until the operation reaches the call to a Provisioner (provisioner operation ID is stored)
-	err := s.poller.Invoke(func() (bool, error) {
-		op, err := s.db.Operations().GetProvisioningOperationByInstanceID(iid)
-		assert.NoError(s.t, err)
-		if op.ProvisionerOperationID != "" {
-			return true, nil
-		}
-		return false, nil
-	})
-	assert.NoError(s.t, err)
-	return s.provisionerClient.LastProvisioning()
-}
-
 func (s *BrokerSuiteTest) Log(msg string) {
 	s.t.Helper()
 	s.t.Log(msg)
 }
 
-func (s *BrokerSuiteTest) EnableDumpingProvisionerRequests() {
-	s.provisionerClient.EnableRequestDumping()
-}
-
 func (s *BrokerSuiteTest) GetInstance(iid string) *internal.Instance {
 	inst, err := s.db.Instances().GetByID(iid)
 	require.NoError(s.t, err)
 	return inst
 }
 
-func (s *BrokerSuiteTest) processProvisioningByOperationID(opID string) {
-	s.WaitForProvisioningState(opID, domain.InProgress)
-	s.AssertProvisionerStartedProvisioning(opID)
-
-	s.FinishProvisioningOperationByProvisionerAndInfrastructureManager(opID, gqlschema.OperationStateSucceeded)
-	_, err := s.gardenerClient.Resource(gardener.ShootResource).Namespace(fixedGardenerNamespace).Create(context.Background(), s.fixGardenerShootForOperationID(opID), v1.CreateOptions{})
-	require.NoError(s.t, err)
-
-	// provisioner finishes the operation
-	s.WaitForOperationState(opID, domain.Succeeded)
-}
-
-func (s *BrokerSuiteTest) processKIMOnlyProvisioningByOperationID(opID string) {
+func (s *BrokerSuiteTest) processKIMProvisioningByOperationID(opID string) {
 	s.WaitForProvisioningState(opID, domain.InProgress)
 
 	s.FinishProvisioningOperationByInfrastructureManager(opID)
 }
 
-func (s *BrokerSuiteTest) processUpdatingByOperationID(opID string) {
-	s.WaitForProvisioningState(opID, domain.InProgress)
-
-	s.FinishUpdatingOperationByProvisioner(opID)
-
-	// provisioner finishes the operation
-	s.WaitForOperationState(opID, domain.Succeeded)
-}
-
-func (s *BrokerSuiteTest) failProvisioningByOperationID(opID string) {
-	s.WaitForProvisioningState(opID, domain.InProgress)
-	s.AssertProvisionerStartedProvisioning(opID)
-
-	s.FinishProvisioningOperationByProvisionerAndInfrastructureManager(opID, gqlschema.OperationStateFailed)
+func (s *BrokerSuiteTest) processKIMProvisioningByInstanceID(iid string) {
+	var runtimeID string
+	err := s.poller.Invoke(func() (done bool, err error) {
+		instance, _ := s.db.Instances().GetByID(iid)
+		if instance.RuntimeID != "" {
+			runtimeID = instance.RuntimeID
+			return true, nil
+		}
+		return false, nil
+	})
+	assert.NoError(s.t, err, "timeout waiting for the runtimeID to be created.")
 
-	// provisioner finishes the operation
-	s.WaitForOperationState(opID, domain.Failed)
+	s.ProcessInfrastructureManagerProvisioningRuntimeResource(runtimeID)
```

Comment:
This call sets Runtime CR as Ready. 
Could we consider using the Ready term in the function name?
e.g. InfrastructureSetsRuntimeResourceReady etc